src/api/libcellml/types.h

```diff
@@ -31,6 +32,14 @@ class Units; /**< Forward declaration of Units class. */
 typedef std::shared_ptr<Units> UnitsPtr; /**< Type definition for shared units pointer. */
 class Variable; /**< Forward declaration of Variable class. */
 typedef std::shared_ptr<Variable> VariablePtr; /**< Type definition for shared variable pointer. */
+class XmlNode; /**< Forward declaration of XmlNode class. */
+typedef std::shared_ptr<XmlNode> XmlNodePtr; /**< Type definition for shared XML node pointer. */
```

Comment:
I'm not quite sure why the XML classes and pointers are being exposed in the libcellml types? Or is this a case of your branch being a commit or two behind mine?